src/utils.js

```diff
@@ -389,3 +394,23 @@ export const safeExecute = (
     logger.warn(message, err);
   }
 };
+
+export const getLumigoEventKey = event => {
+  const noCircularEvent = noCirculars(event);
+  return recursiveGetLumigoEventKey(noCircularEvent);
+};
+
+const recursiveGetLumigoEventKey = noCircularEvent => {
+  let foundValue = undefined;
+  Object.keys(noCircularEvent).some(function(k) {
```

Comment:
Use arrow func (also you can save this to as an own function, right now you create new func for every key)